assets/global.js

```diff
@@ -18,6 +18,7 @@ document.querySelectorAll('[id^="Details-"] summary').forEach((summary) => {
     event.currentTarget.setAttribute('aria-expanded', !event.currentTarget.closest('details').hasAttribute('open'));
   });
 
+  if (summary.closest('header-drawer')) return;
```

Comment:
I'm adding a check here so it doesn't add the event listener on the summary elements from the menu drawer. As it has it's own behaviour that is tackled in the class declaration (onKeyUp(event)). 

But is it good idea to add a check for a specific element within a function that is general like this one. 

I could also remove the IDs on the `<details` elements from the menu drawer and re add an accessibility function in the menu drawer class that would basically do the same as what's happening in this one.